parlai/core/worlds.py

```diff
@@ -145,7 +145,7 @@ def clone(self):
         """
         Create a duplicate of the world.
         """
-        return type(self)(opt=copy.deepcopy(self.opt), agents=None, shared=self.share())
+        return type(self)(opt=self.opt, agents=None, shared=self.share())
```

Comment:
why were we doing this before?

Reply:
Abundance of caution. But the worlds were making their own deep copy on init